app/controllers/api/comments_controller.rb

```diff
@@ -22,7 +22,12 @@ def create
     @comment.user = current_user
     @comment.commentable = commentable
     if @comment.save
-      render :create, status: :created
+      # Inquiry以外はvueでJSONを使用している。コメント機能のJS変換が完了すれば下の'Inquiry'による分岐は不要なので削除してください。
```

Comment:
```suggestion
      # TODO: Inquiry以外はvueでJSONを使用している。コメント機能のJS変換が完了すれば下の'Inquiry'による分岐は不要なので削除してください。
```

`# TODO:`という形式で書くという慣習があるのでこうするといいかも

Reply:
アノテーションコメント、という書き方だと知りました！
勉強になりました。